sdk/communication/Azure.Communication.Chat/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 1.2.0-beta.1 (Unreleased)
 
 ### Features Added
+- Added support for an optional `kind` property for `CommunicationIdentifierSerializer`.
```

Comment:
`CommunicationIdentifierSerializer` is an `internal` class that's not part of the public API and the customers have no knwledge about it. 
if you want to add a changelog, you may choose to add something like "Optimized the logic for deserializing types derived from the `CommunicationIdentifier`".